addons/udes_stock/models/stock_warehouse.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class StockWarehouse(models.Model):
+    _name = "stock.warehouse"
+    _inherit = ["stock.warehouse", "mail.thread"]
+
+    # Add tracking for archiving.
+    active = fields.Boolean(track_visibility="onchange")
+    u_pallet_barcode_regex = fields.Char("Pallet Barcode Format", default="^UDES(\\d)+$")
+    u_package_barcode_regex = fields.Char("Package Barcode Format", default="^UDES(\\d)+$")
```

Comment:
Default for both pallets and packages are `UDES` could that trip us up?

Reply:
In 11.0 these are all set in most customers `warehouse.xml` data file anyway. If we want to change the default, am happy to

Reply:
The idea with most systems we do going forwards though is that we will avoid have a customer module unless really needed. So most likely we would configure this anyway, it just seems having both the same as default seems dangerous. I'll ask around though.

Reply:
Let's do the same as 11 for now, so pallet default `PAL` and package default `PACK`. We can always adjust later if needed.